cfme/services/service_catalogs/ui.py

```diff
@@ -88,7 +88,6 @@ def is_displayed(self):
         return (
             self.in_service_catalogs and
             self.service_catalogs.is_opened and
-            'Service "{}"'.format(self.context['object'].name) in self.title.text and
```

Comment:
Is there no other check for this page? breadcrumb? Otherwise how does the user (much less our automation) know that they're about to order the correct service?